spec.md

```diff
@@ -2982,36 +2538,30 @@ A multiplexer expression is legal only if the following holds.
 
 1. The types of the two input expressions are equivalent.
 
-1. The types of the two input expressions are passive (see
-   [@sec:passive-types]).
+1. The types of the two input expressions are passive (see [@sec:passive-types]).
 
 ## Primitive Operations
 
-All fundamental operations on ground types are expressed as a FIRRTL primitive
-operation. In general, each operation takes some number of argument expressions,
-along with some number of integer literal parameters.
+All fundamental operations on ground types are expressed as a FIRRTL primitive operation.
+In general, each operation takes some number of argument expressions, along with some number of integer literal parameters.
 
 The general form of a primitive operation is expressed as follows:
 
 ``` firrtl
 op(arg0, arg1, ..., argn, int0, int1, ..., intm)
 ```
 
-The following examples of primitive operations demonstrate adding two
-expressions, `a`{.firrtl} and `b`{.firrtl}, shifting expression `a`{.firrtl}
-left by 3 bits, selecting the fourth bit through and including the seventh bit
-in the `a`{.firrtl} expression, and interpreting the expression `x`{.firrtl} as
+The following examples of primitive operations demonstrate adding two expressions, `a`{.firrtl} and `b`{.firrtl}, shifting expression `a`{.firrtl} left by 3 bits, selecting the fourth bit through and including the seventh bit in the `a`{.firrtl} expression, and interpreting the expression `x`{.firrtl} as
 a Clock typed signal.
 
-``` firrtl
+```firrtl
```

Comment:
Nit: unrelated, but I'll allow it. 😉 This will get cleaned up when we run the formatter.